app/controllers/static_pages_controller.rb

```diff
@@ -10,4 +10,7 @@ def help; end
 
   # GET static_pages/about.
   def about; end
+
+  # GET static_pages/contact.
```

Comment:
`static_pages` -> `/static_pages`, Also, you need the second "human" comment for these controller actions. Same for whole file.